src/tests/multihost/playbook/build_go_ssh_package.yaml

```diff
@@ -0,0 +1,18 @@
+---
+- hosts: localhost
+  vars:
+    repo_loc: "{{ twd }}/../sssd"
+  gather_facts: False
+  tasks:
+    - name: build a go package before test on agent
+      shell: |
+          git clone https://github.com/appleboy/easyssh-proxy.git
+          pushd easyssh-proxy # i replaced cd with pushd seems prettier
+          go get -u github.com/appleboy/easyssh-proxy
+          go build -buildmode=c-shared -o /tmp/ssh_library.so {{ repo_loc }}/src/tests/multihost/sssd/testlib/common/remote_ssh.go
+          chmod 777 /tmp/ssh_library.so
+          cp -vf /tmp/ssh_library.so  {{ repo_loc }}/src/tests/multihost/sssd/testlib/common/
+          popd
```

Comment:
Can you split this up into separate ansible tasks per step in the shell code?

Reply:
Done